src/components/Dropdown.js

```diff
@@ -0,0 +1,37 @@
+import { useState } from 'react'
+import styled from "styled-components";
+import Input from './Input.js';
+
+const DropdownSelect = styled.select`
+font-size: ${(p) => p.size ? p.theme.fontSizes[p.size] : p.theme.fontSizes.default};
+font-family: ${(p) => p.theme.fonts};
+font-weight: ${(p) => p.theme.fontWeight};
+color: ${(p) => p.theme.colors.text};
+padding: 10px;
+border-radius: 5px;
+margin: 0.25em;
+padding: 0.25em 0.25em;
+`;
+
+const Dropdown = ({ options, defaultOption, size, showInput, setShowInput, customValue, ...props }) => {
+    const [selected, setSelected] = useState(defaultOption);
+    return (
+        <div>
+            <DropdownSelect size={size} value={selected} onChange={(e) => handleSelect(e, setSelected, setShowInput, customValue)}>
```

Comment:
Update the handleSelect param in response to the handleSelect method changing places